docker-compose.yml

```
@@ -0,0 +1,57 @@
+      - redis_db
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: django-mindjunkies
+    environment:
+      - PORT=8000
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    entrypoint: [ "sh", "-c", "chmod +x /app/migrate.sh /app/entrypoint.sh && /app/migrate.sh && /app/entrypoint.sh" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)